dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -108,12 +110,18 @@ public int createPullReq(GHRepository origRepo, String branch,
             log.warn("Handling error with pull request creation... {}", e.getMessage());
             JsonElement root = JsonParser.parseString(e.getMessage());
             JsonArray errorJson = root.getAsJsonObject().get("errors").getAsJsonArray();
-            String error = errorJson.get(0).getAsJsonObject().get("message").getAsString();
+            String error = "";
+            if (errorJson.get(0).getAsJsonObject().has("message")) {
```

Comment:
Store the output of `errorJson.get(0).getAsJsonObject()` as a local variable.  Call `has` and `get` on that variable.  This removes the need to call `get(0).getAsJsonObject()` twice.

Reply:
Why are we ignoring other errors if multiple were returned?